types/rangy/rangy-classapplier.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for Rangy Class Applier module
+// Project: https://github.com/timdown/rangy
+// Definitions by: Maxime Kjaer <https://github.com/MaximeKjaer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+///<reference path="index.d.ts"/>
+
+interface RangyStatic {
+    createClassApplier(theClass: string, options?: RangyClassApplierOptions, tagNames?: Array<string> | string): RangyClassApplier;
+}
+
+interface RangyClassApplierOptions {
+    elementTagName?: string;
+    elementProperties?: {[property: string]: any};
+    elementAttributes?: {[attribute: string]: any};
+    ignoreWhiteSpace?: boolean;
+    applyToEditableOnly?: boolean;
+    tagNames?: Array<string> | string;
+    normalize?: boolean;
+    onElementCreate?: (element: Element, classApplier: RangyClassApplier) => void;
+    useExistingElements?: boolean;
+}
+
+interface RangyClassApplier extends RangyClassApplierOptions {
```

Comment:
The return type of `getNativeTextRange()` seems to be [well defined](https://github.com/timdown/rangy/wiki/Rangy-Selection#getnativetextrange) for example.